### PR TITLE
Hello microverse Project

### PR DESCRIPTION
**Pull request by prolajumokeoni**
In this project I:
- Set up a linter for HTML and CSS and make sure that any files that are not relevant are ignored by git
- Added index.html, style.css, and test.md file
- Added a .gitignore file
- Added README file

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link rel="stylesheet" href="style.css">
+  <title>Hello Microverse</title>
+</head>
+<body>
+  <p>Hello Microverse</p>
```
Comment (Donard97) [inline index.html]: - [x] Please change the paragraph Hello Microverse to an h1 header because it was one of the requirements! 💪

File: style.css [hunk 2]
```diff
@@ -0,0 +1,4 @@
+p {
+  color: pink;
+  font-weight: bolder;
+}
```
Comment (Donard97) [inline style.css]: - [x] Kindly change the tag "p" to "h1" after you change the HTML because the style will not work. ✌️

**Review by Donard97 — CHANGES_REQUESTED** [review]
## Hi @prolajumokeoni , 

Good job so far! 💯 

There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights

- Gitflow is correctly used ✔️ 
- The readme file is professional ✔️ 
- PR is descriptive ✔️ 
- No linter errors ✔️ 

## Required Changes ♻️

- [x] Kindly add test.md file to .gitignore file because it should be ignored

**_Check the comments under the review._**

### Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

**Cheers and Happy coding!👏👏👏**

_Feel free to leave any questions or comments in the PR thread if something is not 100% clear._

Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

**Review by Donard97 — APPROVED** [review]
# Hi @prolajumokeoni ,  ✋ 


Good job on implementing the required changes 🎉🎉🎉

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 

## Congratulations! 🎉 

giphy (5)


**Cheers and Happy coding!👏👏👏**

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

File: index.html (hunk 1 above)
Comment (prolajumokeoni) [reply index.html]: Thankyou for the review, implemented all the required changes

**Pull request merged**